crates/napi/src/next_api/project.rs

```diff
@@ -402,6 +404,12 @@ pub async fn project_new(
         memory_limit,
         dependency_tracking,
     )?;
+
+    turbo_tasks.send_compilation_event(Arc::new(DiagnosticEvent::new(
+        "Starting the compilation events server...".into(),
```

Comment:
Small nit: I prefer a slightly more explicit conversion function when available.

```suggestion
        "Starting the compilation events server...".to_owned(),
```